### ISSUE TITLE
Search among the saved tabs for duplicates, and delete them

### ISSUE THREAD

**Issue by cxw42** [cxw42/TabFern#320]
### Discussed in https://github.com/cxw42/TabFern/discussions/319

<div type='discussions-op-text'>

<sup>Originally posted by **Alunar05** July 31, 2023</sup>
... So far, there are 2 features that I believe would be incredibly useful: ...

2. A feature to (search and) delete duplicates 
</div>

**Comment by Darthfett** [comment]
+1 to this.  I have a bad habit of re-opening links to the same things, and Tabfern does seem positioned well to help me with finding these doppelgangers, so I can get rid of them, and keep them organized.

**Comment by cxw42** [comment]
@Alunar05 @Darthfett are you referring to duplicates within a window?  Across windows?

**Comment by Alunar05** [comment]
Hello. Either way. I just want to have saved any website only once. The plugin in its function automatically saves (and in this process, it duplicates) the sets of tabs that are currently open. I suppose this happens when the browser or the PC are restarted. In this process, the plugin has created automatically dozens of duplicates of everything

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Chris White ***@***.***>
Enviado: Saturday, November 23, 2024 8:04:46 PM
Para: cxw42/TabFern ***@***.***>
Cc: Alunar05 ***@***.***>; Mention ***@***.***>
Asunto: Re: [cxw42/TabFern] Search among the saved tabs for duplicates, and delete them (Issue #320)


@Alunar05<https://github.com/Alunar05> @Darthfett<https://github.com/Darthfett> are you referring to duplicates within a window? Across windows?

**Comment by Darthfett** [comment]
@cxw42 I personally don't want the same tab in two places across all windows.  When opening a tab from a link, it just ends up in whatever the last active window was, so only looking in the same window isn't nearly as helpful.

**Comment by cxw42** [comment]
ACK --- thanks!  I'll need to think about this a bit.  Determining whether one window is a subset or another should at least be possible.
